### DISCUSSION TITLE
Error installing on macOS arm64

### DISCUSSION THREAD

**Discussion by rockingh0rse** [Homebrew/discussions#464] [Getting started]
Hi,
As the M1 hasn't been out long, just thought I'd mention this as it may not have been spotted yet:

Installation worked fine but got this error message right at the end...

```
/bin/bash: line 680: shell_profile: unbound variable
==> Next steps:
- Run `brew help` to get started

```

Regards,
Stuart

**Comment by SMillerDev** [comment]
Please create an issue in https://github.com/homebrew/install

**Comment by carlocab** [comment]
Opened a PR: https://github.com/Homebrew/install/pull/521
